tests/python/rest_api/test_projects.py

```diff
@@ -475,6 +475,22 @@ def test_can_import_export_dataset_with_imagenet_format(self):
 
         self._test_import_project(username, project_id, format_name, import_data)
 
+    def test_can_import_export_dataset_with_pascal_voc_format(self):
```

Comment:
Since #4544 is merged, you should be able to roll this test into the preceding one (by adding another value for the parameter).

Reply:
Updated